### PR TITLE
[16.0][MIG] rma_repair

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
Migration to 16.0

Depends on:
* [x] rma_account #397 

@ForgeFlow

### CODE REVIEW

File: rma_repair/views/rma_order_line_view.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                >
+                        <field
+                        name="repair_count"
+                        widget="statinfo"
+                        string="Repair Orders"
+                    />
+                </button>
+            </div>
+            <group name="quantities" position="inside">
+                <group>
```
Comment (OriolMForgeFlow) [inline rma_repair/views/rma_order_line_view.xml]:
```suggestion
                <group attrs="{'invisible': [('repair_type', '=', 'no')]}">
```
Only show the quantities if some Repair Policy has been selected (similar to the receipt_policy, deliver_policy or sales_policy).

File: rma_repair/views/rma_order_line_view.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        <field name="inherit_id" ref="rma.view_rma_line_form" />
+        <field name="arch" type="xml">
+            <div name='button_box' position="inside">
+                <button
+                    type="object"
+                    name="action_view_repair_order"
+                    class="oe_stat_button"
+                    icon="fa-wrench"
+                    groups="stock.group_stock_user"
+                >
```
Comment (OriolMForgeFlow) [inline rma_repair/views/rma_order_line_view.xml:17]:
```suggestion
                    attrs="{'invisible':[('repair_count','=',0)]}"
                >
```

Hide the smart button if there are no repair orders.

Comment (DavidJForgeFlow) [reply rma_repair/views/rma_order_line_view.xml:17]: Done!

File: rma_repair/views/rma_order_line_view.xml (hunk 1 above)
Comment (DavidJForgeFlow) [reply rma_repair/views/rma_order_line_view.xml]: Done!

**Pull request merged**